x-pack/plugins/apm/public/components/shared/KueryBar/view.js

```diff
@@ -112,16 +114,25 @@ class KueryBarView extends Component {
             style={{ display: 'inline-block', marginTop: '10px' }}
             title={
               <div>
-                There&#39;s no APM index pattern with the title &#34;
-                {apmIndexPatternTitle}
-                &#34; available. To use the Query bar, please choose to import
-                the APM index pattern via the{' '}
-                <KibanaLink
-                  pathname={'/app/kibana'}
-                  hash={`/home/tutorial/apm`}
-                >
-                  Setup Instructions.
-                </KibanaLink>
+                <FormattedMessage
+                  id="xpack.apm.kueryBar.indexPatternMissingWarningMessage"
+                  defaultMessage="There's no APM index pattern with the title &#34;{apmIndexPatternTitle}&#34; available.
+                    To use the Query bar, please choose to import the APM index pattern via the {setupInstructionsLink}."
```

Comment:
Is it Github's syntax highlighting that's off, or should 119 and 120 be on a single line?

Reply:
I also wonder if we could use a template literal so we don't have to use html entities like `&#34;`

Reply:
> Is it Github's syntax highlighting that's off, or should 119 and 120 be on a single line?

I think so. I broke the text into 2 line just for better readability.

We can not use a template literal here. I moved quotes to `values`, please take a look
https://github.com/elastic/kibana/blob/d35cabd59b986d3a9c26a087e2a11eeaf00dae64/x-pack/plugins/apm/public/components/shared/KueryBar/view.js#L119-L121